### ISSUE TITLE
Python wrapped version results in warnings with Python 3.9.4

### ISSUE THREAD

**Issue by hjmjohnson** [InsightSoftwareConsortium/ITKTextureFeatures#93]
Python 3.9.4 
```bash
pip install itk -U
pip install itk-texturefeatures
```
ITK 5.2.0 python install + itk-texturefeatures 3.5.0 install.

```python
help(itk.Image)
```

results in the following warning:
```
itkFixedArrayF8 not loaded from module TextureFeatures because of exception:
 module 'itk.TextureFeaturesPython' has no attribute 'itkFixedArrayF8'
itkFixedArrayF10 not loaded from module TextureFeatures  because of exception:
 module 'itk.TextureFeaturesPython' has no attribute 'itkFixedArrayF10'
```

**Comment by bpaniagua** [comment]
I just saw this as well.

**Comment by PranjalSahu** [comment]
I don't get this warning with itk 5.3.0.

**Comment by PranjalSahu** [comment]
Ok the description in the issue is not full.
I get the warning when I call `help(itk.TextureFeatures)` not `help(itk.Image)`

**Comment by PranjalSahu** [comment]
I see that itkVector and itkFixedArray have been wrapped for itkRunLengthTextureFeaturesImageFilter.
itkVector has been used here at [itkRunLengthTextureFeaturesImageFilter.wrap](https://github.com/InsightSoftwareConsortium/ITKTextureFeatures/blob/master/wrapping/itkRunLengthTextureFeaturesImageFilter.wrap#L56) but itkFixedArray is not used. So I will make a build and test by adding itkFixedArray also in this file.

**Comment by bpaniagua** [comment]
Fantastic, thank you! Please, keep us posted!

**Comment by PranjalSahu** [comment]
Fixed in https://github.com/InsightSoftwareConsortium/ITKTextureFeatures/pull/98

**Comment by thewtex** [comment]
Released in 3.6.0: https://pypi.org/project/itk-texturefeatures/#history
